api/app/tests/requests/test_pteams.py

```diff
@@ -1826,6 +1231,75 @@ def test_get_pteam_members__by_not_member():
     assert response.reason_phrase == "Forbidden"
 
 
+def test_update_pteam_members():
+    create_user(USER1)
+    user2 = create_user(USER2)
+    pteam1 = create_pteam(USER1, PTEAM1)
+    invitation = invite_to_pteam(USER1, pteam1.pteam_id)
+    accept_pteam_invitation(USER2, invitation.invitation_id)
+
+    request = {"is_admin": True}
+    response = client.put(
+        f"/pteams/{pteam1.pteam_id}/members/{user2.user_id}", headers=headers(USER1), json=request
+    )
+    assert response.status_code == 200
+    assert response.json()["pteam_id"] == str(pteam1.pteam_id)
+    assert response.json()["user_id"] == str(user2.user_id)
+    assert response.json()["is_admin"] == request["is_admin"]
+
+
+def test_update_pteam_members__by_not_admin():
```

Comment:
```suggestion
def test_it_should_return_403_when_noadmin_update_pteam_members():
```
可能であればテスト名から試していることがわかる様にして欲しいです